packages/monorepo-builder/packages/Merge/ComposerJsonDecorator/RemoverComposerJsonDecorator.php

```diff
@@ -119,5 +119,9 @@ private function processRoot(ComposerJson $composerJson, ComposerJson $removingC
         if ($removingComposerJson->getPreferStable()) {
             $composerJson->removePreferStable();
         }
+
+        if (count($removingComposerJson->getRepositories()) === 1 && $removingComposerJson->getRepositories()[0] === '*') {
+            $composerJson->setRepositories([]);
+        }
```

Comment:
Thanks for this feature!

`*` usually means include everything, so I would expect it to merge all the packages. "Remove include everything" is too many negative at once.

Could we use constant here to make the feature move obvisou? E.g. `public const REMOVE_COMPLETELY` or something similar.

Reply:
> ...follow the same logic as in the example where 'phpunit/phpunit' => '*' is removed

Hm :thinking: That sounds confusing. Maybe that would deserve some UX upgrade to more clear text as well.

> Your proposal would be to include REMOVE_COMPLETELY in the Option class? Something like this:

Yes, exactly :+1: Thank you